src/org/thoughtcrime/securesms/database/MmsDatabase.java

```diff
@@ -921,6 +924,7 @@ public long insertMessageOutbox(@NonNull MasterSecretUnion masterSecret,
     contentValues.put(ADDRESS, message.getRecipient().getAddress().serialize());
     contentValues.put(DELIVERY_RECEIPT_COUNT, Stream.of(earlyDeliveryReceipts.values()).mapToLong(Long::longValue).sum());
     contentValues.put(READ_RECEIPT_COUNT, Stream.of(earlyReadReceipts.values()).mapToLong(Long::longValue).sum());
+    contentValues.put(HASHED_ID, generateHashedId(message.getSentTimeMillis()));
```

Comment:
What happens if you send a multi length message (160+ characters)? Will this still work based off of the sent time?

Reply:
Is a multi length message a message that gets auto sent as two+ bubbles?